aioshelly/const.py

```diff
@@ -70,6 +70,7 @@
     "SNSW-001X16EU": "Shelly Plus 1",
     "SNSW-0024X": "Shelly Plus I4",
     "SNSW-002P16EU": "Shelly Plus 2PM",
+    "SNSW-102P16EU": "Shelly Plus 2PM",  # hw v2
```

Comment:
Where does the information that this is version 2 hardware come from?

Reply:
> Where does the information that this is version 2 hardware come from?

I took inspiration from the "Shelly Button1" where I found 2 entries for the same product.
```
    "SHBTN-1": "Shelly Button1",
    "SHBTN-2": "Shelly Button1",  # hw v2
```
Then I replicated it without doing any re-search and gave no more thoughts to this.
I understand now that this was not the right idea.

The Shelly Plus 2PM device I added was bought in Norway if that is of any help.
I can try to gather more info about the device and find out why there exists an extra model name for it.

Reply:
We are waiting for an answer from Shelly dev team whether it is really a new hardware version.

Reply:
Great!
Don't know if it is of any help, but here is the JSON output from my shelly plus 2pm device when requesting the **http://<host>/shelly** endpoint.
```
name: "shelly_alias"
id: "shellyplus2pm-441793cf094c"
mac: "441793CF094C"
model: "SNSW-102P16EU"
gen: 2
fw_id: "20221206-141957/0.12.0-gafc2404"
ver: "0.12.0"
app: "Plus2PM"
auth_en: true
auth_domain: "shellyplus2pm-441793cf094c"
profile: "switch"
```